internal/command/admin/deploy/deploy.go

```diff
@@ -20,16 +20,55 @@ import (
 )
 
 type Deploy struct {
-	Docker     deploy_docker     `cmd:"" name:"docker" aliases:"d" help:"deploy locally with 1 core and 1 worker docker containers"`
-	Kubernetes deploy_kubernetes `cmd:"" name:"kubernetes" aliases:"k,k8s" help:"deploy on an existing kubernetes cluster"`
+	Docker     deploy_docker     `cmd:"" name:"docker" aliases:"d" help:"Deploy locally with 1 core and 1 worker docker containers"`
+	Kubernetes deploy_kubernetes `cmd:"" name:"kubernetes" aliases:"k,k8s" help:"Deploy on an existing kubernetes cluster"`
 }
 
 type deploy_docker struct {
-	Up   docker.Up   `cmd:"" name:"up" aliases:"u" help:"spin up Docker-based FunLess deployment"`
-	Down docker.Down `cmd:"" name:"down" aliases:"d" help:"tear down Docker-based FunLess deployment"`
+	Up   docker.Up   `cmd:"" name:"up" aliases:"u" help:"Spin up Docker-based FunLess deployment"`
+	Down docker.Down `cmd:"" name:"down" aliases:"d" help:"Tear down Docker-based FunLess deployment"`
 }
 
 type deploy_kubernetes struct {
-	Up   kubernetes.Up   `cmd:"" name:"up" aliases:"u" help:"spin up Kubernetes-based FunLess deployment"`
-	Down kubernetes.Down `cmd:"" name:"down" aliases:"d" help:"tear down Kubernetes-based FunLess deployment"`
+	Up   kubernetes.Up   `cmd:"" name:"up" aliases:"u" help:"Spin up Kubernetes-based FunLess deployment"`
+	Down kubernetes.Down `cmd:"" name:"down" aliases:"d" help:"Tear down Kubernetes-based FunLess deployment"`
+}
+
+func (f *Deploy) Help() string {
+	return `
+Below is a description of the architecture.
+
+CORE
+
+	The orchestrator of the platform: it manages functions and modules.
+	In case of invocation requests, it acts as a scheduler to pick one 
+	of the available worker components.
+	
+WORKER
+
+	The actual functions executor: it is able to run multiple functions 
+	in parallel. In case of invocation requests, it uses a cache to avoid 
+	requesting the same function multiple times.
+
+PROMETHEUS
+
+	Collects metrics from the platform containers (Core and Worker).
+	It is also used by the Core to access the metrics of the Workers 
+	to make scheduling decisions. 
+
+POSTGRES
+
+	Used by the Core to store functions and modules.
+
+ELASTICSEARCH
+KIBANA
+FILEBEAT
+
+	This stack is used in docker deploy to collect and manage logs. 
+
+`
+}
+
+func (f *deploy_docker) Help() string {
```

Comment:
I would remove this one as it is just a repetition of the standard help message.